Dropp/Dropp-swift/Dropp/UserObject.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  UserObject.swift
+//  Dropp
+//
+//  Created by Pongsakorn Cherngchaosil on 4/9/17.
+//  Copyright © 2017 Group B. All rights reserved.
+//
+
+import Foundation
+
+class UserObject : NSObject {
+    var userId: String?
+    var location: String?
+    var timestamp: String?
+    var content: String?
```

Comment:
We only need to have media and text strings. I think this content should be media for right now